cmd/xl-v1-bucket.go

```diff
@@ -136,7 +137,7 @@ func (xl xlObjects) getBucketInfo(ctx context.Context, bucketName string) (bucke
 	// reduce to one error based on read quorum.
 	// `nil` is deliberately passed for ignoredErrs
 	// because these errors were already ignored.
-	readQuorum := len(xl.getDisks()) / 2
+	readQuorum := int(math.Ceil(float64(len(xl.getDisks())) / 2.0))
```

Comment:
```suggestion
	readQuorum := readQuorum(len(xl.getDisks())
```

(or just inline below)